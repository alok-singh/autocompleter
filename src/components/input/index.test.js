import React from 'react';
import { render } from '@testing-library/react';
import Input from './index';

test('Unit tests for Input component', () => {
  expect(render(<Input 
    value="some text"
    onChange={() => {}}
  />).baseElement).toMatchSnapshot();
});
