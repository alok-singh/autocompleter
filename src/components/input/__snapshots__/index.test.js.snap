// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit tests for Input component 1`] = `
<body>
  <div>
    <input
      class="u-input"
      value="some text"
    />
  </div>
</body>
`;
